test/functional/mapreduce.test.js

```diff
@@ -386,7 +330,7 @@ describe('MapReduce', function () {
     }
   });
 
-  it('shouldCorrectlyReturnNestedKeys', {
+  it.skip('shouldCorrectlyReturnNestedKeys', {
```

Comment:
@mbroadst Do you think this test should be rewritten without `collection.group`, or is it fine to just remove it outright:?

Reply:
I think it's fine to remove outright. It kind of looks like this test is testing the behavior of MongoDB, rather than some behavior of the driver itself